realtime/utils.js

```diff
@@ -47,8 +52,12 @@ function getNewObjAsString(key, obj) {
  * @param {Object}  messgObj - Message object received from the redis channel.
  * @returns {Object} - returns the parsed message object.
  */
-function parseObject(messgObj) {
-  if (messgObj.new) {
+function parseObject(messgObj, key) {
+  if (key === eventName.del && messgObj.new) {
+    return messgObj.old;
+  }
+
+  if (key === eventName.add && messgObj.new) {
     return messgObj.new;
   }
 
```

Comment:
You might want to add some comments here to say why we ended by doing like this. Also, you can combine these "if" conditions to use else if's and else.